### PR TITLE
Update documentation to include bindings

### PR DESCRIPTION
**Pull request by mapalan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,27 +32,28 @@ The `RedisPubSubTrigger` subscribes to a Redis pub/sub channel and surfaces mess
...
+> This trigger is not capable of listening to keyspace notifications on clustered caches.
+
 #### Inputs
 - `ConnectionStringSetting`: Name of the setting in the appsettings that holds the to the Redis cache connection string (eg `<cacheName>.redis.cache.windows.net:6380,password=...`).
 - `Channel`: pubsub channel that the trigger should listen to.
   - Supports channel patterns.
-  - This field can be resolved using `INameResolver`.
 
-#### Avaiable Output Types
+#### Avaiable Parameter Types
```
Comment (MSFTeegarden) [inline README.md:43]: FYI, "Available" is misspelled as "Avaiable" in all these headers :)

File: README.md [hunk 2]
```diff
@@ -123,8 +123,54 @@ public static void StreamsTrigger(
 }
 ```
 
-## Known Issues
-- The `RedisPubSubTrigger` is not capable of listening to [keyspace notifications](https://redis.io/docs/manual/keyspace-notifications/) on clustered caches.
```
Comment (MSFTeegarden) [inline README.md:127]: Is this not true anymore?

File: README.md [hunk 3]
```diff
@@ -123,8 +123,54 @@ public static void StreamsTrigger(
...
+### `Redis` Input Binding
+
+#### Inputs
+- `ConnectionStringSetting`: Name of the setting in the appsettings that holds the to the Redis cache connection string (eg `<cacheName>.redis.cache.windows.net:6380,password=...`).
+- `Command`: The redis-cli command to be executed on the cache with all arguments separated by spaces.
+
+> **Note**
+> Not all commands are supported for this binding. At the moment, only read commands that return a single output are supported.
```
Comment (MSFTeegarden) [inline README.md:133]: Can we list a few more examples of what type of commands are supported and what types are not? That will help people have a better understanding. Once we've settled on a set of commands that we feel good about, we should list them all.

File: README.md [hunk 4]
```diff
@@ -123,8 +123,54 @@ public static void StreamsTrigger(
...
+- [`StackExchange.Redis.RedisValue`](https://github.com/StackExchange/StackExchange.Redis/blob/main/src/StackExchange.Redis/RedisValue.cs), `string`, `byte[]`, `ReadOnlyMemory<byte>`: The value returned by the command.
+- `Custom`: The trigger uses Json.NET serialization to map the value returned by the command from a `string` into a custom type.
+
+#### Sample
+The following gets any key that was recently set.
+```c#
+[FunctionName(nameof(SetGetter))]
+public static void SetGetter(
+    [RedisPubSubTrigger("Redis", "__keyevent@0__:set")] string key,
+    [Redis("Redis", "GET {Message}")] string value,
```
Comment (MSFTeegarden) [inline README.md:145]: How do we handle parameters that can be passed in with commands? Is everything stuffed into the command, or are they passed in through other means?

File: README.md [hunk 5]
```diff
@@ -123,8 +123,54 @@ public static void StreamsTrigger(
...
+#### Function Return type
+- `string[]`: Arguments for the redis command.
+
+#### Sample
+The following deletes any key that was recently set.
+```c#
+[FunctionName(nameof(SetDeleter))]
+public static void SetDeleter(
+    [RedisPubSubTrigger("Redis", "__keyevent@0__:set")] string key,
+    [Redis("Redis", "DEL")] out string[] arguments,
```
Comment (MSFTeegarden) [inline README.md:167]: Likely dumb question: what are some examples of arguments being returned? For instance, `DEL` returns the number of keys deleted. Is that what would be returned here?

Comment (mapalan) [reply README.md:167]: In the case of output bindings, nothing is returned to the customer's function
The `DEL` command only runs after the function is finished

File: README.md (hunk 4 above)
Comment (mapalan) [reply README.md:147]: For multiple parameters, it would be `HGET key field` with spaces in between each value

File: README.md (hunk 2 above)
Comment (MSFTeegarden) [reply README.md:127]: Nope, you're good. I just totally missed it.

File: README.md [hunk 6]
```diff
@@ -89,19 +91,18 @@ public static void ListsTrigger(
...
+- `ConnectionStringSetting`: Name of the setting in the appsettings that holds the to the Redis cache connection string (e.g. `<cacheName>.redis.cache.windows.net:6380,password=...`).
 - `Key`: Key to read from.
-  - This field can be resolved using `INameResolver`.
 - `PollingIntervalInMs`: How often to poll Redis in milliseconds.
   - Default: `1000`
 - `MaxBatchSize`: Number of entries to read from Redis at one time. These are processed in parallel.
   - Default: `16`
 
-#### Avaiable Output Types
+#### Avaiable Parameter Types
```
Comment (MSFTeegarden) [inline README.md:105]: Available is still misspelled here :)

File: README.md [hunk 7]
```diff
@@ -123,8 +123,55 @@ public static void StreamsTrigger(
...
-- The `RedisPubSubTrigger` is not capable of listening to [keyspace notifications](https://redis.io/docs/manual/keyspace-notifications/) on clustered caches.
+### `Redis` Input Binding
+
+#### Inputs
+- `ConnectionStringSetting`: Name of the setting in the appsettings that holds the to the Redis cache connection string (e.g. `<cacheName>.redis.cache.windows.net:6380,password=...`).
+- `Command`: The redis-cli command to be executed on the cache with all arguments separated by spaces. (e.g. `"GET key"`, `"HGET key field"`)
+
+> **Note**
+> Not all commands are supported for this binding. At the moment, only read commands that return a single output are supported.
+> The full list can be found [here](./src/Microsoft.Azure.WebJobs.Extensions.Redis/Bindings/RedisConverter.cs#L61)
```
Comment (MSFTeegarden) [inline README.md:134]: This is awesome. Can we do the same thing for output bindings?

**Review by MSFTeegarden — APPROVED** [review]

**Pull request merged**